chromium_src/extensions/common/constants.cc

```diff
@@ -10,6 +10,8 @@ const int kBraveActionRightMargin = 2;
 
 const char brave_extension_id[] = "mnojpmjdmbbfmejpflffifhffcmidifd";
 const char brave_rewards_extension_id[] = "jidkidbbcafjabdphckchenhfomhnfma";
+const char file_type_policies_extension_id[] =
```

Comment:
just use a static const in https://github.com/brave/brave-core/pull/6763/files#diff-388a0bd24beb5682ff5395e49b02229fd9216b6c216ae24a5a8c4ed6a47d596dR17 because we shouldn't depend on extensions